### PR TITLE
Raise missing ValueError in transform_angles

### PR DESCRIPTION
**Pull request by has2k1**
- Includes a small PEP8 change
- Added tests to cover the `Transform.transform_angles` method

### CODE REVIEW

File: lib/matplotlib/tests/test_transforms.py [hunk 1]
```diff
@@ -535,6 +536,21 @@ def test_nan_overlap():
...
 
+def test_transform_angles():
+    t = mtrans.Affine2D()
+    angles = np.array([20, 45, 60])
+    points = np.array([[0, 0], [1, 1], [2, 2]])
+
+    new_angles = t.transform_angles(angles, points)
+    assert_array_almost_equal(angles, new_angles)
+
+    with assert_raises(ValueError) as cm:
```
Comment (WeatherGod) [inline lib/matplotlib/tests/test_transforms.py]: for future coders, could you add a comment explaining the error you are triggering?

**Comment by WeatherGod** [comment]
I restarted the failing test, now it is saying that `assert_raises()` takes three arguments... did nose bork a release or something?

**Comment by jenshnielsen** [comment]
@WeatherGod Assert_raises can't be used as a context manager in python 2.6. I would suggest just skipping this test on python 2.6 as done in https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/tests/test_rcparams.py#L127

**Comment by WeatherGod** [comment]
Didn't know that. Good to know!

**Comment by has2k1** [comment]
Well, that worked this time around.

Commented the tests.

**Comment by WeatherGod** [comment]
This does not need backporting. The exception was only missing in master.

**Comment by tacaswell** [comment]
@has2k1 Thanks! I should have caught that.

**Pull request merged**